Orso.Arpa.Api/Program.cs

```diff
@@ -46,6 +48,25 @@ public static IHostBuilder CreateHostBuilder(string[] args) =>
                         _ = logging.ClearProviders();
                         _ = logging.SetMinimumLevel(LogLevel.Trace);
                     })
+                    .ConfigureServices((context, services) =>
```

Comment:
das hier alles bitte wieder rausnehmen. Wir konfigurieren CORS in Startup.cs und das soll unbedingt in der prod Umgebung laufen. Wenn du da was separates für den Raspi brauchst, dann gerne auf einem separaten Branch aber ohne PR. Dein yml File dann so ausführen, dass es auf deinem Branch ausgeführt wird.

Reply:
Ahhh alles klar, wird gemacht. Ich war sowieso nicht ganz sicher wo das plötzlich herkam. Habe Program.cs wirklich nicht angefasst.. ick schwööööre 😆  Hatte es dringelassen, weil ich dachte, daß das evtl. irgend n Automatismus ist wenn man neuen workflow anlegt oder so. Ist mir schleierhaft wie das kommt. Im Develop-Branch war das auch schon, ich habs reverted. Dann neuen branch (also diesen) gemacht, die Datei angelegt und schwups waren diese neuen Zeilen in Program.cs schon wieder drinne wie von Geisterhand 🤔